about.html

```diff
@@ -0,0 +1,164 @@
+              <li id="bordered-menu-item"><a href="">Buy a ticket</a></li>
+            </ul>
+
+            <section class="mobile-only" id="ham">
+              <img src="./assets/ham.png" alt="" onclick="showMobileMenu()" />
+            </section>
+          </nav>
+        </header>
+
+        <p class="about-1-title mobile-only">"Hello! Cinema lovers"</p>
+        <h1 class="about-2-title">Iranian film week<br />in finland 2023</h1>
+        <hr class="logo-part-line1 desktop-only" />
+        <p class="about-p">
+          A survey from 150 Iranian critics and professionals showed in the best
+          Persian movie magazine, World of Picture, Bashu was voted as “The Best
+          Iranian Film of All Time”. According to the credits, It was the film’s
+          ability to promote social commentary in a subversive government that
+          made it noteworthy for all times.
+        </p>
+        <h2 class="about-3-title">
+          Please contact us per Email for any further questions about<br />Iranian
+          film week in Finland 2023!
+        </h2>
+        <h4 class="about-4-title">
+          <a href="mailto:film@finland.fi">film@finland.fi</a>
+        </h4>
+      </section>
+
+      <section class="logo-part">
+        <h3 class="main-program-title">The National Museum Logo</h3>
+        <hr class="logo-part-line1" />
+        <h2 class="about-3-title">
+          The logo of the national museum was designed<br />by John Doe in 1998
+        </h2>
+        <img class="logo-box" src="./assets/logo-black.png" />
+        <hr class="logo-part-line2" />
+        <h3 class="main-program-title">See the film weeks in Finland</h3>
+        <hr class="logo-part-line1" />
+        <h2 class="about-3-title">
+          Take a look at the last two film weeks<br />which took place in Oulu
+          and Tampere
+        </h2>
+
+        <section class="about-images">
+          <img src="./assets/11.jpg" alt="" class="red-image" />
+          <img src="./assets/22.jpg" alt="" class="red-image" />
+        </section>
+      </section>
+
+      <section class="partners-container mobile-only" id="sponsor">
+        <h3 class="main-program-title">Partner</h3>
+        <hr class="main-program-line" />
+        <section class="partners">
+          <img src="./assets/cann.png" alt="" />
+          <img src="./assets/imdb.png" alt="" />
+          <img src="./assets/bbc.png" alt="" />
+          <img src="./assets/oscars.png" alt="" />
+        </section>
+      </section>
+
+      <section class="footer-about">
+        <img
+          src="./assets/logo-white.png"
+          alt=""
+          class="footer-about-logo desktop-only"
+        />
+
+        <img
+          src="./assets/logo-black.png"
+          alt=""
+          class="footer-about-logo mobile-only"
+        />
+        <h2 class="footer-about-text">
+          2023 film week in Finland<br />Some rights reserved.
+        </h2>
+      </section>
```

Comment:
- [ ] Kindl make sure to wrap your footer in the `footer` tag always in order to structure your HTML semantically correctly. :wink: